pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -373,12 +373,12 @@ func satisfyPodAffinity(state *preFilterState, nodeInfo *framework.NodeInfo) boo
 // It checks if a pod can be scheduled on the specified node with pod affinity/anti-affinity configuration.
 func (pl *InterPodAffinity) Filter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod, nodeInfo *framework.NodeInfo) *framework.Status {
 	if nodeInfo.Node() == nil {
-		return framework.NewStatus(framework.Error, "node not found")
+		return framework.AsStatus(fmt.Errorf("node not found"))
```

Comment:
Note: this is fine to move to AsStatus or leave it as it was. Since there is no wrapping, we are not losing any information.

Reply:
Agree, will rollback such changes since they do not makes things better.